src/components/Company/AssignSignatory/InviteSignatory/InviteSignatory.tsx

```diff
@@ -60,18 +57,11 @@ export function InviteSignatory(props: InviteSignatoryProps & BaseComponentInter
   )
 }
 
-function Root({
-  companyId,
-  signatoryId,
-  defaultValues,
-  className,
-  children,
-}: InviteSignatoryProps) {
+function Root({ companyId, defaultValues, className, children }: InviteSignatoryProps) {
   useI18n('Company.AssignSignatory')
   const { onEvent, baseSubmitHandler } = useBase()
 
   const { data: signatories } = useGetAllSignatories(companyId)
-  const currentSignatory = signatories.find(signatory => signatory.uuid === signatoryId)
```

Comment:
Cleanup: currentSignatory was leftover from before a refactor. it doesn't actually get used so removed